### ISSUE TITLE
zowe-install-apf-server.yaml TSS Facility Owner not documented

### ISSUE THREAD

**Issue by John-A-Davies** [zowe/docs-site#727]
## Description

The entries for these lines in `zowe-install-apf-server.yaml` are not documented.
```
# TSS Facility Owner (Required for TSS. 'auto' supplies the running user)
  tssFacilityOwner=auto
```

**Comment by nannanli** [comment]
Reviewed the `zowe-install-apf-server.yaml` and found that another parameter in this file is not documented either. The parameter was added by Vitek in Aug. 
```
# ZSS server name (default name is ZWESIS_STD),
  # make sure that this equals to zssCrossMemoryServerName in zlux-server in zowe-install.yaml
  zssCrossMemoryServerName=ZWESIS_STD
```
@VitekVlcek-Broadcom For the parameter above, could you provide a description about it and anything that users need to do about this configuration? This is the [documentation](https://docs.zowe.org/stable/user-guide/configure-zowe-runtime.html#scripted-install-of-the-zowe-cross-memory-server) for the `zowe-install-apf-server.yaml` file which might help you better understand what's needed. Just let me know if any questions. Thanks!

**Comment by VitekVlcek-Broadcom** [comment]
Thank you @nannanli for spotting it, we prepared some time ago a pull request regarding this
https://github.com/zowe/docs-site/pull/581.
TLDR: if you want to run multiple different version of Zowe in parallel, each zowe instance must have unique `zssCrossMemoryServerName`. If user wants to run only one version of zowe, than default name is sufficient. If they want to test a new version of zowe in parallel to an older version of zowe, they have to install new zowe with unique `zssCrossMemoryServerName` ie change default name.

Until zowe 1.5 user had to change `zssCrossMemoryServerName` manually,  PR https://github.com/zowe/docs-site/pull/581 describes how to do it. Since 1.5 user can use  `zssCrossMemoryServerName` option in yaml file.
Does this help?
